MobileBuy/buy/src/main/java/com/shopify/buy/dataprovider/BuyRetrofitService.java

```diff
@@ -88,28 +88,25 @@
      * Checkout API
      */
 
-    @POST("anywhere/checkouts.json")
+    @POST("api/checkouts.json")
```

Comment:
lol, this endpoint has been flipped back and forth between api and anywhere about 5 times in our prs.  Not sure which branch is the culprit but anywhere keeps sneaking back in :)